stdlib/public/core/Range.swift

```diff
@@ -447,6 +470,22 @@ extension PartialRangeUpTo: RangeExpression {
   }
 }
 
+extension PartialRangeUpTo: Codable where Bound: Codable {
+  private enum CodingKeys: String, CodingKey {
+    case upperBound = "beginningUpTo"
```

Comment:
For consistency with "toEndFrom", I think this would be "fromStartUpTo" (and similarly below, "fromStartThrough").